src/PlusCommon/PlusCommon.h

```diff
@@ -47,13 +47,13 @@ enum PlusImagingMode
 
 /* Define case insensitive string compare for Windows. */
 #if defined( _WIN32 ) && !defined(__CYGWIN__)
-#  if defined(__BORLANDC__)
-#    define STRCASECMP stricmp
-#  else
-#    define STRCASECMP _stricmp
-#  endif
+  #if defined(__BORLANDC__)
+    #define STRCASECMP stricmp

```

Comment:
I know. The issue is that there are multiple case-insensitive string comparison implementation in PlusCommon for us to use, maintain, document. We should keep only one.